modules/core/context.js

```diff
@@ -25,7 +25,7 @@ import { utilKeybinding, utilRebind, utilStringQs, utilCleanOsmString } from '..
 
 export function coreContext() {
   const dispatch = d3_dispatch('enter', 'exit', 'change');
-  let context = utilRebind({}, dispatch, 'on');
+  const context = {};
```

Comment:
This works because JS autocompletion is powered by TypeScript, which is smart enough to infer the type of `context` when defined as an empty object

[proof here](https://www.typescriptlang.org/play/?filetype=js#code/MYewdgzgLgBAHjAvDA3gXwNwCgtwHQCGSMAFAJRIB8MAjLngEbHlUwBEb2WMP+BWAegE8YAPQD8g4bwDkEGFACeABwCmAE1UAzGAEt5usFtUAnExpw4h1m7bv2HNy1i0BXMMCi7wMAO4ALAihVADdTFhQYcyhXEzBUNBg0HFBIWEViAKDQ8LJsIRExSSxFQmYKRGo6UqZkFkr2ThwRUv4CngkpFrkFFQ1tGGACMBlYBlU9I1NzdRgsIA)

utilRebind can safely be moved to the `return` statement because this class does not use `context.on` internally.